test/core/consensus/grandpa/vote_graph/ghost_introduce_branch_test.cpp

```diff
@@ -127,11 +127,20 @@ TEST_F(VoteGraphFixture, GhostIntroduceBranch) {
 })");
   }
 
+  {
+    auto ghostOpt =
+        graph->findGhost(boost::none, [](auto &&x) { return x >= "5"_W; });
+    ASSERT_TRUE(ghostOpt);
+    ASSERT_EQ(*ghostOpt, BlockInfo(10, "ED"_H))
+        << "The farthest block of blocks with enough weight should be selected";
```

Comment:
[non-critical]
The **best** block of blocks with enough weight should be selected

Reply:
fixed